package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "website",
+  "version": "1.0.0",
+  "description": "This is a preliminary setup for our seads Frontend website, which will eventually include d3 visualizations. Currently, this website will only work with a local postgres database, which can be installed with the instructions below.",
```

Comment:
Where are the instructions?

See my other comment--no need to create the database manually.